packages/insomnia-app/app/main/window-utils.js

```diff
@@ -56,6 +59,7 @@ export function createWindow() {
     height: height || DEFAULT_HEIGHT,
     minHeight: MINIMUM_HEIGHT,
     minWidth: MINIMUM_WIDTH,
+    titleBarStyle: titleBarStyle,
```

Comment:
Could you please favor ES6 features for this (sorry my OCD :) )

```suggestion
    titleBarStyle,
```